src/libstd/fs.rs

```diff
@@ -975,13 +975,18 @@ impl FileType {
 
     /// Test whether this file type represents a symbolic link.
     ///
+    /// The Metadata struct needs to be retreived with
```

Comment:
This place wants more inline-code annotations, so it looks like

> retrieved with `fs::symlink_metadata()` not `fs::metadata()`.

and not

> retrieved with fs::symlink_metadata() not fs::metadata().

You can achieve that enclosing the inline code snippets within the grave (`) symbols. Like this:

```markdown
retrieved with `fs::symlink_metadata()` not `fs::metadata()`.
```